### ISSUE TITLE
Create a Cleanup Schedule for Unused/ Retired AWS Accounts

### ISSUE THREAD

**Issue by MaxWardle** [bcgov/cloud-pathfinder#2167]
**Describe the Issue**
We have a lot of accounts/ project sets in AWS that we're paying minimum $50/ month for but are not being used. We can close 10% of all accounts within a rolling 30 day window for each landing zone/ ECF so we should set up a schedule to remove account when we can.

**Additional Context**
- "Sandbox" accounts should be closed once we move to "Tools"
  - there are 62 Sandbox accounts in ECF Live
- Each account in the SEA runs about $50/ month baseline (nothing deployed except for SEA architecture) 
- Over 200 accounts deployed in ECF Live alone
- There are "Retired" project sets that teams have used and then moved off the platform
- There are project sets for testing landing zone upgrades (used once and then never again)
- There are 20 lab accounts in ECF Live alone, of which we're using less than half
- There are many accounts also in ECF Forge and ECF Assembly that should be removed
  - The rolling 30 day window is separate for these landing zones so we could remove accounts in parallel
- Some accounts are marked for deletion in the "notes" meta field defined in the project.json

**Acceptance Criteria**
- determine what account we want to close
  - meeting with tech team and PO to discuss costs and priorities
- create docs for closing accounts if they do not already exist
- create a 30 day schedule/ reminder to delete project sets (automated ticket creation?)

.
.
.
.
.
.
.
.
.
.
.

**Extra Template Info**

***Definition of Ready***
These set of conditions will need to be met in order to bring a ticket into the sprint and start work. Protects the team from unclear requirements.
Issues (Task/Story/Spike) aligned to an EPIC and linked appropriately.
Assigned to the appropriate CPF MEMBER (and is not left blank in ZenHub) at the start of sprint (1st day of the sprint)
Acceptance has been defined for the issue and has been reviewed with CPF team and approved by the necessary approver.
Has been sized and estimated by the delivery team.
Detailed breakdown of the steps required to complete the story in the additional context
Any additional specifications have been documented, reviewed with CPF and approved by the necessary approver.

***Acceptance Criteria***
A set of pre-defined requirement that need to be met in order to mark the user story as “done”.
It should be testable with no room for interpretation
It should be either “pass” or “fail”
It should be clear enough for business stake holders to understand
As part of the user story, it should be written from the user perspective
A well written acceptance criteria is great for 
- managing expectations
- defining scope
- reducing ambiguity
- establishing testing criteria for QA
Given (Pre-condition) 
When (Action)
Then (Outcome)

***Definition of Done***
These set of conditions are met for work to be considered as complete on an issue.
All acceptance criteria(s) have been validated.
All the necessary documentation for the issue has been uploaded to Teams.
Functionality has been tested when applicable.
Functionality has been demonstrated to the relevant stakeholders (where applicable).
Story has been scheduled for a Sprint Demo/community update and impacted teams invited to the Demo.
Stories accepted by PO and documented for potential sharing with impacted users.
- Documented = Captured in Community Update (CU) deck
- Release notes = aspirational goal of providing a bulleted list of features and changes that our users can touch. Was going to store them in our Cloud Pathfinder repo. Can also display to users in the CU

**Comment by wrnu** [comment]
no longer relevant, accounts have been cleaned
